<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RTShortcutDialog</class>
 <widget class="QDialog" name="RTShortcutDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>180</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Assign keyboad shortcut</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::InputKeyboard"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="gbFncKeys">
     <property name="title">
      <string>Key Modifier</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="spacing">
       <number>12</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="cbxKeyShift">
        <property name="text">
         <string>Shift</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="cbxKeyControl">
        <property name="text">
         <string>Control</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="cbxKeyAlt">
        <property name="text">
         <string>Alt</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cbxKeyMeta">
        <property name="text">
         <string>Meta</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="cbxKeyPad">
        <property name="text">
         <string>Keypad</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QKeySequenceEdit" name="edKeyShortcut">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
     <property name="maximumSequenceLength">
      <longLong>1</longLong>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edKeyShortcut</tabstop>
  <tabstop>cbxKeyShift</tabstop>
  <tabstop>cbxKeyControl</tabstop>
  <tabstop>cbxKeyAlt</tabstop>
  <tabstop>cbxKeyMeta</tabstop>
  <tabstop>cbxKeyPad</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RTShortcutDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RTShortcutDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
